sample_project/Assets/SampleViewer/Samples/FeatureLayer/Scripts/FeatureLayer.cs

```diff
@@ -196,6 +200,7 @@ private void CreateFeatures(int min, int max)
                         currentFeature.properties.propertyNames.Add(props[0]);
                         currentFeature.properties.data.Add(props[1]);
                         featureInfo.Properties.Add(key);
+                        featureInfo.Index = j;
```

Comment:
Can you replace `I` and `j` with a word/label that describes what they are?

Reply:
I don't think this is super necessary. Any user that wants to look at code and has knowledge of for loops will expect the variables to be i and j for each respective loop. Changing the variable names isn't going to make impact the sample or readability of the code significantly so I don't think it is necessary.

Reply:
I dont think using single letter var names is appropriate when its an easy fix, ten minutes to rename, to change it to something readable for the user. In this case, `j` is actually a `featureInfoIndex`.

If anything, using single letter var names makes the code less clear. Its on me, but it took me a second to figure out what the `j` stood for. I'm pretty sure @airaolagoitia would say the same, since I used to get dinged by him for it too back in the day haha. We can't use them in the plugin code, and we should maintain the same standard in the sample code, since it's an extension of the plugin. While rare, single letter var names could easily clash with other similar var names, so its better to be clear and descriptive to make the code more robust and easier to read.

This is just me too, I feel using them also harkens back to the old school days of programming (well before our time haha) when its was more pragmatic to use single letter var names because of file size, less robust tools, etc. We have the power now haha.

And to be clear, I'm not asking just to be nitpicky, I genuinely feel that, in general, avoiding using single letter var names and using clear descriptive names, even in for loops, leads to better code.